kornia/augmentation/augmentation.py

```diff
@@ -141,7 +148,8 @@ class RandomVerticalFlip(AugmentationBase2D):
                   [0., 0., 0.]]]]), tensor([[[ 1.,  0.,  0.],
                  [ 0., -1.,  2.],
                  [ 0.,  0.,  1.]]]))
-
+        >>> seq.inverse(seq(input)[0]).equal(input)
```

Comment:
why do you need to index ?

Reply:
since the ```return_transform```. Yeah, a good point. Should be fixed.